packages/app/providers/auth-provider.tsx

```diff
@@ -98,25 +100,9 @@ export function AuthProvider({
           if (Platform.OS !== "web") {
             router.pop();
           }
-
           setTimeout(() => {
-            router.push(
-              Platform.select({
-                native: "/profile/complete",
-                web: {
-                  pathname: router.pathname,
-                  query: {
-                    ...router.query,
-                    completeProfileModal: true,
-                  },
-                } as any,
-              }),
-              Platform.select({
-                native: "/profile/complete",
-                web: router.asPath,
-              })
-            );
-          }, 100);
+            router.push("/profile/complete-prompt");
```

Comment:
IMO we should get rid of the way before, using this will be more concise and clear.